week1/leftRightScrollBtn.js

```diff
@@ -0,0 +1,13 @@
+document.addEventListener("DOMContentLoaded", () => {
+  let scrollContainer = document.querySelector(".group");
+  let scrollLeftBtn = document.getElementById("scrollLeftBtn");
+  let scrollRightBtn = document.getElementById("scrollRightBtn");
+
+  scrollLeftBtn.addEventListener("click", () => {
+    scrollContainer.scrollLeft -= 2000;
+  });
+
+  scrollRightBtn.addEventListener("click", () => {
+    scrollContainer.scrollLeft += 2000;
+  });
```

Comment:
이 버튼은 `scrollWidth()`랑 `clientWidth()`를 활용해서 최대 너비 값만큼 스크롤을 이동시켜서 오른쪽 끝에 붙게 구현해봤어!
> scrollWidth: scroll할 수 있는 숨겨진 영역을 모두 합한 width
cliendtWidth: border나 scrollbar를 포함하지 않은 width


```suggestion
  scrollRightBtn.addEventListener("click", () => {
    scrollContainer.scrollLeft = scrollContainer.scrollWidth - scrollContainer.clientWidth;
  });
```

Reply:
아아아아아아아 이런 방법이..!